lib/modules/speaker/SpeakerAccessory.js

```diff
@@ -91,16 +91,10 @@ class SpeakerAccessory extends GenericAccessory {
   /*----------========== HOMEBRIDGE STATE SETTERS/GETTERS ==========----------*/
 
   getVolumeOn() {
-    // if (this.isMiotDeviceConnected()) {
-    //   return this.getDevice().getMuteValue() ? false : true;
-    // }
-    return this.isMiotDeviceConnected();
+    return this.isMiotDeviceConnected() ? true : false;
```

Comment:
This should already return a boolean so I do not think that it should be necessary?

Reply:
I add a log:
`this.logger.warn('this.isMiotDeviceConnected()=%o', this.isMiotDeviceConnected(), 'type=', typeof this.isMiotDeviceConnected())`

and the result is here:
```
[2022/11/22 17:29:33] [miot] [小爱音响Pro] this.isMiotDeviceConnected()={
  did: 'xxx',
  token: 'xxx',
  longitude: '0.0',
  latitude: '0.0',
  name: 'Mi AI Speaker Pro',
  pid: '0',
  localip: 'xxx',
  mac: 'xxx',
  ssid: 'xxx',
  bssid: 'xxx',
  parent_id: '',
  parent_model: '',
  show_mode: 1,
  model: 'xiaomi.wifispeaker.lx06',
  adminFlag: 1,
  shareFlag: 0,
  permitLevel: 16,
  isOnline: true,
  desc: 'Device online ',
  extra: {
    isSetPincode: 0,
    pincodeType: 0,
    fw_version: '1.82.5',
    needVerifyCode: 0,
    isPasswordEncrypt: 0
  },
  uid: 14388109,
  pd_id: 1233,
  password: '',
  p2p_id: '',
  rssi: -44,
  family_id: 0,
  reset_flag: 0
} type= object
```

Reply:
I think the reason is because `BaseAccessory.isMiotDeviceConnected` => `BaseDevice.isConnected` => `MiotDevice.isConnected` => `MiotDevice.isConnectedToCloud` => `MiotDevice.getCloudDeviceInfo` returns an object.